pkg/addons/default/coredns.go

```diff
@@ -1,245 +0,0 @@
-package defaultaddons
-
-import (
-	"fmt"
-	"strings"
-	"time"
-
-	"github.com/kris-nova/logger"
-	"github.com/pkg/errors"
-
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	apierrs "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/apimachinery/pkg/watch"
-
-	"github.com/weaveworks/eksctl/pkg/kubernetes"
-)
-
-const (
-	// CoreDNS is the name of the coredns addon
-	CoreDNS = "coredns"
-	// KubeDNS is the name of the kube-dns addon
-	KubeDNS = "kube-dns"
-
-	componentLabel = "eks.amazonaws.com/component"
-
-	coreDNSImagePrefix = "602401143452.dkr.ecr."
-	coreDNSImageSuffix = ".amazonaws.com/eks/coredns"
-)
-
-// InstallCoreDNS will install the `coredns` add-on in place of `kube-dns`
-func InstallCoreDNS(rawClient kubernetes.RawClientInterface, region, controlPlaneVersion string, waitTimeout *time.Duration, plan bool) (bool, error) {
```

Comment:
We only want to get rid of this function.

Reply:
Change reverted, and `UpdateCoreDNS` kept.